raiden/tests/utils/blockchain.py

```diff
@@ -25,6 +26,7 @@
 DEFAULT_BALANCE = denoms.ether * 10000000
 DEFAULT_BALANCE_BIN = str(denoms.ether * 10000000)
 DEFAULT_PASSPHRASE = 'notsosecret'  # Geth's account passphrase
+GIGA = 1024 * 1024 * 1024
```

Comment:
Maybe let's give the exact size for the DAG of the 0 epoch?

It is exactly `1073739912` bytes.